sale_generator/models/sale_order.py

```diff
@@ -12,4 +11,3 @@ class SaleOrder(models.Model):
     generator_id = fields.Many2one(
         comodel_name="sale.generator", string="Generator"
     )
-    is_template = fields.Boolean()
```

Comment:
Not sure it should be removed. But probably we could replace this field by an active set to True in this case.
Then template exist from generator but not from sales without to have to hack sale.order domain action

cc @sebastienbeau

Reply:
Yes, not sure either, @sebastienbeau could you help us understand how the client uses this, how do we solve the problem of differentiating whether SOs are used as templates or not and how to find them ? Does the client want to know ? If yes, how should they go about it ?

In my opinion, if they only want to use the generator as a simple copy-paste, they shouldn't care about it... but because we keep the generators and their source SO, they might misuse it (by trying to generate more copies of an outdated SO) and create confusion.

If they want to really track generators and SOs used as templates for the generators, then we need a more elaborate solution and menu items with filters etc

Reply:
The new version uses is_template field again as well as active field to help separate sale order templates from sale orders